issue 180, clarification of quotation in "Filter" Parameter (#752)
MicrosoftDocs/windows-powershell-docs: docset/windows/addsadministration/get-aduser.md

For a list of supported types for \<value\>, type `Get-Help about_ActiveDirectory_ObjectModel`.

> Note: As type of the parameter is "String", PowerShell will cast the filter query to a string while processing this command. It means, that in case of using variable as a value in the filter component, make sure that it complies with the [PowerShell Quoting Rules](https://docs.microsoft.com/powershell/module/microsoft.powershell.core/about/about_quoting_rules). For example, if filter expression is double-quoted, variable should be enclosed using single quotation marks: **Get-ADUser -Filter "Name -like '$UserName'"**. On the contrary, if curly braces are used to enclose the filter, variable shouldn't be quoted at all: **Get-ADUser -Filter {Name -like $UserName}**.

Reviewer: ```suggestion
Note: For String parameter type, PowerShell will cast the filter query to a string while processing the command. When using a string variable as a value in the filter component, make sure that it complies with the [PowerShell Quoting Rules](https://docs.microsoft.com/powershell/module/microsoft.powershell.core/about/about_quoting_rules). For example, if the filter expression is double-quoted, the variable should be enclosed using single quotation marks: **Get-ADUser -Filter "Name -like '$UserName'"**. On the contrary, if curly braces are used to enclose the filter, the variable should not be quoted at all: **Get-ADUser -Filter {Name -like $UserName}**.
```
